testnet/stacks-node/conf/testnet-follower-conf.toml

```diff
@@ -8,11 +8,11 @@ wait_time_for_microblocks = 10000
 [burnchain]
 chain = "bitcoin"
 mode = "xenon"
```

Comment:
Oh, I just noticed, I think since this is a regtest network now, this should be krypton.

Reply:
@obycode Does it just make sense to consolidate the testnet and regtest follower confs? The configuration behind the two are pretty much identical now if not exactly identical. The main designation now is just what the network operator _chooses_ to call it based on how it's used.

Reply:
Yeah, I think it probably does.